Python/pytime.c

```diff
@@ -1,6 +1,7 @@
 #include "Python.h"
 #ifdef MS_WINDOWS
 #include <windows.h>
+#include <winsock2.h>
```

Comment:
Would you mind to add a comment to explain why we need to import networking functions in pytime.c? :-) (To get the timeval structure)